### ISSUE TITLE
indent rule for """ text blocks

### ISSUE THREAD

**Issue by Cyberbeni** [nicklockwood/SwiftFormat#360]
Swiftformat indents texts inside """ blocks.

**Comment by nicklockwood** [comment]
@cyberbeni that sounds like a pretty serious bug. Do you have a code sample I can use to reproduce it?

**Comment by Cyberbeni** [comment]
https://github.com/WeTransfer/WeScan

format WeScan/Extensions/CIImage+Utils.swift
the indentation of the last """ gets changed
modify the file elsewhere and format again
the indentation inside the block changes

**Comment by nicklockwood** [comment]
@Cyberbeni OK, I can confirm this is a bug. Thanks for reporting it. Minimum repro case seems to be:

```swift
foo(baz:
    """
    baz
    """)
```

For now, the workarounds I would suggest are to either 1) add the following swiftformat directives to the file:

```swift
// swiftformat:disable:indent,wrapArguments
guard let colorKernel = CIColorKernel(source:
    """
    kernel vec4 color(__sample pixel, float inputEdgeO, float inputEdge1)
    {
        float luma = dot(pixel.rgb, vec3(0.2126, 0.7152, 0.0722));
        float threshold = smoothstep(inputEdgeO, inputEdge1, luma);
        return vec4(threshold, threshold, threshold, 1.0);
    }
    """
    ) else { return nil }
// swiftformat:enable:indent,wrapArguments
```

Or 2) manually reformat the function as:

```swift
guard let colorKernel = CIColorKernel(source: """
kernel vec4 color(__sample pixel, float inputEdgeO, float inputEdge1)
{
    float luma = dot(pixel.rgb, vec3(0.2126, 0.7152, 0.0722));
    float threshold = smoothstep(inputEdgeO, inputEdge1, luma);
    return vec4(threshold, threshold, threshold, 1.0);
}
""") else { return nil }
```

which doesn't trigger the bug.

**Comment by nicklockwood** [comment]
@Cyberbeni fixed in 0.39.1.
